docs/reference/release-process.md

```diff
@@ -3,78 +3,81 @@ title: Release Process
 ---
 
 ## Release Owner
+
```

Comment:
Most of these changes occured automatically upon editing/saving this file in VS Code. The only explicit changes I made were replacing instances of `git pull` with `git pull --tags`.

Reply:
Tags are fetched by default  (see `--no-tags` on https://git-scm.com/docs/git-pull).  The change here doesn't hurt but also not totally necessary.

Reply:
Thanks Barry; it's a little confusing why they have both a `--tags` and a `--no-tags` option, but I guess people can have different configs to change default behavior and then want to override things for example in specific instances.

If it does the same thing; it doesn't make sense to merge this. @bbangert : Is it possible something else is going on then for the discrepancies in the 225.2 dot release?

Reply:
I have verified that `git pull` does indeed pull tags by default. So I'm going to close this PR. Thanks @chenba for the review and for helping verify with my test repo.